src/interface/sparse_pack_base.cpp

```diff
@@ -138,7 +144,11 @@ SparsePackBase SparsePackBase::Build(T *pmd, const PackDescriptor &desc,
 
   // Allocate the views
   int leading_dim = 1;
-  if (desc.with_fluxes) {
+  pack.flx_idx_ = 1;
+  if (contains_face_with_fluxes) {
+    leading_dim += 5;
```

Comment:
5 because of the face itself and 4 edges/fluxes?

Reply:
`leading_dim` starts at 1, so this makes it 6. When a face has fluxes, the face itself has 3 components and the edge flux has 3 components.